cla-backend/cla/utils.py

```
@@ -2056,7 +2056,7 @@ def extract_pull_request_number(pull_request_message):
             cla.log.debug(f"{fn} - extracted PR number {pull_request_number} from merge_queue data: {pull_request_message} by matching first '#N'")
             return pull_request_number
         else:
-            cla.log.warning(f"{fn} - error - unable to extract pull request number from message: {pull_request_message}")
-    except Exception as e:
-        cla.log.warning(f"{fn} - error - unable to extract pull request number from message: {pull_request_message}, error: {e}")
+            cla.log.warning(f"{fn} - error - unable to extract pull request number from message")
+    except (ValueError, json.JSONDecodeError, AttributeError, KeyError) as e:
```

Ruff (0.15.5)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)